src/components/ResourceSettingsModal.jsx

```diff
@@ -246,7 +247,16 @@ export default class ResourceSettingsModal extends Component {
               <p className={elementStyles.formLabel}>Resource Type</p>
 
               {/* Permalink or File URL */}
-              <button type="button" onClick={this.permalinkFileUrlToggle}>{permalink ? 'Switch to download' : 'Switch to post'}</button>
+              <div className="d-flex">
+                <label htmlFor="radio-post" className="flex-fill">
+                  <input type="radio" id="radio-post" name="resource-type" value="post" onClick={this.handlePermalinkFileUrlToggle} checked={!!permalink} />
```

Comment:
Is there a particular reason why you're using `!!`?

Reply:
It's more of a cleanliness thing as I wanted to make it clear that the `checked` attribute works like a boolean